plugins/@grouparoo/facebook/src/lib/export/utils.ts

```diff
@@ -0,0 +1,332 @@
+/**
```

Comment:
What's different here than the facebook lib?  Could we patch it vs re-implemeting?

Reply:
for some reason, it was splitting first and last name it to 5 characters. I think there will be a few other changes too.